Week1/hw4/accountBook.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>당니의 가계부</title>
+</head>
+<body>
+    <h1 class="title">당니의 가계부</h1>
+    <section class="myStatus">
+        <p class="total_txt">나의 자산</p>
+        <p class="total">341200</p>
+        <div class="totalIncomeCost">
+            <img src="./images/plus.png" class="plus" />
+            <p class="income" style="display:inline;">341200&nbsp;&nbsp;&nbsp;&nbsp;</p>
+            <img src="./images/minus.png" class="minus"/>
+            <p class="cost" style="display:inline;">50300</p>
+        </div>
+    </section>
+    <br>
+    <section>
+        <article class="dateArea">
+            <div class="selectDate">
+                <input type="button" class="left_btn">
+                <p class="date">10월 1일</p>
+                <input type="button" class="right_btn">
+            </div>
+            <p class="listText">내역 리스트</p>
+            <input 
+                type="checkbox" 
+                id="income"
+                checked="checked">
```

Comment:
```suggestion
            <input 
                type="checkbox" 
                id="income"
                checked>
```

checked는 boolean값이라 이렇게만 주셔도 된답니당!